plugin.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "emr-clusters",
-    "version": "0.0.2",
+    "version": "0.0.3",
```

Comment:
please do not change the version within PR here as we mill  do in during  publication